scripts/base/files/x509/main.zeek

```diff
@@ -101,7 +101,17 @@ export {
 	## to work cluster wide, but come at a slightly higher cost of memory and inter-node-communication.
 	##
 	## This setting is ignored if Zeek is run in standalone mode.
```

Comment:
Should we perhaps add a note here that this is ignored if known_log_certs_enable_publish is set to true?